monai/networks/utils.py

```diff
@@ -310,3 +312,76 @@ def train_mode(*nets: nn.Module):
         # Return required networks to eval_list
         for n in eval_list:
             n.eval()
+
+
+def compatible_mod_state(
+    dst: Union[torch.nn.Module, Mapping],
+    src: Union[torch.nn.Module, Mapping],
+    dst_prefix="",
```

Comment:
that will make the API complicated and I don't find a use case for now

Reply:
there might be a case to remove certain strings from `src_keys` to match the `dst`, but it'll just need the length of the `src_prefix`. it could be confusing so I don't add it for now

Reply:
Or maybe you can extend the `mapping` arg to handle the prefix problem? Then we can also remove this `dst_prefix` arg.
For example: `mapping={"XXX": "", "": "YYY", "layer_1.weight": "layer.weight"}`
Where `XXX` and `YYY` can be prefix to add/remove for all the state key.
What do you think?

Thanks.

Reply:
No I'll keep the current API simple for now